lua/entities/gmod_wire_expression2/core/player.lua

```diff
@@ -465,6 +465,10 @@ e2function number entity:inNoclip()
 	return 1
 end
 
+e2function number entity:inGodMode()
+	return IsValid(this) and this:HasGodMode() and 1 or 0
```

Comment:
p.s. I am leaving out the `Entity:IsPlayer()` check, since `this` entity should be either a player or NULL now.

Reply:
Um, damn E2, what's the point of separating it as `player.lua` other than *more organized code*.